src/constants.js

```diff
@@ -83,6 +83,21 @@ export const hridSettingsSections = [
   },
 ];
 
+export const holdingsStatementTypes = [
+  {
+    type: 'holdingsStatement',
+    title: 'Holdings statement',
+  },
+  {
+    type: 'holdingsStatementForSupplements',
+    title: 'Holdings statement for supplements',
+  },
+  {
+    type: 'holdingsStatementForIndexes',
+    title: 'Holdings statement for indexes',
+  },
+];
```

Comment:
Shouldn't these `title` values be translation keys like `ui-inventory.holdings.statement` etc?

Reply:
We use the value `title` only as a key in objects of the corresponding `Holdings statement` type. In other words, we don't use its value as a translation key for output to the UI.

Does it make sense in this case to use the `intl` library?

Reply:
Ah, OK. I misunderstood how `title` was being used.